api/libs/Dockerfile.dal

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0
```

Comment:
This provides an image to run the database migration project.  The only issue is that it doesn't output the migration output for some reason.  So you can't tell if it fails or is successful.